app/blog/page.tsx

```
@@ -5,6 +5,8 @@ import { BlogListing } from '@/components/blog/BlogListing';
 import Link from 'next/link';
 
 export const metadata = { title: 'CloudFix Blog' };
+export const revalidate = 3600; // 1 hour ISR
+export const revalidate = 3600; // 1 hour ISR per Phase 5
```

Biome (2.1.2)

[error] Shouldn't redeclare 'revalidate'. Consider to delete it or rename it.

'revalidate' is defined here:


(lint/suspicious/noRedeclare)